client/src/components/ActiveChat/ReceiverMessage/ReceiverMessage.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box, Typography } from '@material-ui/core';
+import useStyles from './useStyles';
+
+interface Props {
+  text: string;
+}
+
+function ReceiverMessage({ text }: Props): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.root}>
```

Comment:
You can pass props to the Box component to user flexbox

Eg: `<Box display="flex" alignItems="center"></Box>`